### PR TITLE
add task solution

### PR DESCRIPTION
**Pull request by purpleFade**
- [DEMO LINK](https://purpleFade.github.io/layout_moyo-header/)
- [TEST REPORT LINK](https://purpleFade.github.io/layout_moyo-header/report/html_report/)

### CODE REVIEW

File: src/style.css [hunk 1]
```diff
@@ -1 +1,63 @@
 /* Styles go here */
```
Comment (OleziO) [inline src/style.css:1]: Remove all comments.

File: src/style.css [hunk 2]
```diff
@@ -1 +1,63 @@
...
+.nav__link[data-qa="hover"] {
+  color: #00acdc;
+}
+
+.is-active {
+  color: #00acdc;
+}
+
+.nav__link:hover {
+  color: #00acdc;
```
Comment (OleziO) [inline src/style.css:49]: Move all repeatable colors to CSS variables.

Comment (OleziO) [reply src/style.css:49]: [More info](https://developer.mozilla.org/en-US/docs/Web/CSS/Using_CSS_custom_properties)

**Review by OleziO — CHANGES_REQUESTED** [review]
Make font size smaller.

**Comment by purpleFade** [comment]
- [DEMO LINK](https://purpleFade.github.io/layout_moyo-header/)
- [TEST REPORT LINK](https://purpleFade.github.io/layout_moyo-header/report/html_report/)

**Pull request closed without merging**